clients/src/main/java/org/apache/kafka/common/network/KafkaChannel.java

```diff
@@ -147,6 +153,7 @@ public KafkaChannel(String id, TransportLayer transportLayer, Supplier<Authentic
         this.disconnected = false;
         this.muteState = ChannelMuteState.NOT_MUTED;
         this.state = ChannelState.NOT_CONNECTED;
+        this.isInterBrokerListener = isInterBrokerListener;
```

Comment:
How about `privilegedListener`?  Then we could add a comment explaining what that means:
1. if there is a control plane listener, only it is privileged
2. otherwise, if there is an inter broker listener, only it is privileged
3. otherwise, all listeners are privileged